wiki/Interface/ko.md

```diff
@@ -1,269 +1,317 @@
----
-outdated: true
----
-
-**들어가기 전에**
-이 문서는 영문문서를 번역 및 내용 추가한 문서 입니다.(b20140320.4 버전 기준)
-이 문서에는 오탈자 및 실제 유저가 사용하지 않는 단어가 있을 수 있습니다. 찾으면 수정해주세요.
-위키 문서를 잘 작성하지 못하는 사람이 작성했습니다. 잘못된 부분이 있으면 수정해주세요.
-<img src="Intro_static.jpg" title="fig:Right|시작화면" alt="Right|시작화면" width="400" /> \_\_TOC\_\_
-= **메인메뉴** = <img src="Introscreen.jpg" title="fig:메인 메뉴" alt="메인 메뉴" width="280" />
-
-1. 오스!로고는 재생되고 있는 노래의 설정된 [BPM에](KO:Timing "wikilink") 맞취 뜁니다(둠칫둠칫!). 노래가 재생되고있지 않으면, 기본설정대로 느리게 뜁니다.
-
--   2. Play를 누르거나 \[P\] 오스! 로고를 클릭하면 플레이 메뉴로로 넘어갑니다.
--   3. Edit를 누르면 \[E\] [에디터를](KO:Editor "wikilink") 위한 선곡화면으로 이동합니다.
--   4. Option을 누르면 \[O\] [설정화면이](KO:Options "wikilink") 왼쪽에서 튀어나옵니다.
--   5. Exit를 누르면 \[Esc\] Osu!를 종료합니다.
--   6. 무작위하게 뽑힌 팁이 메뉴 아래에 나타납니다 잠시 표시되다가 사라집니다.
--   7. 좌측하단의 조그마하게 있는 글자들을 클릭하면 Osu!공식홈페이지로 이동합니다.
--   8. Bancho!와의 연결작업현황을 나타냅니다. 연결되면 사라집니다.
--   9. 우측하단의 버튼들은 [채팅컨트롤입니다](KO:Internet_Relay_Chat "wikilink"). 채팅창을 띄우거나(오른쪽)\[F9\] 확장 채팅 인터페이스(왼쪽)\[F8\]를 띄워 접속중인 유저현황을 볼 수 있습니다. 둘은 항상 같이 나타나지만 채팅창을 내려야 모두 내릴 수 있습니다(유저현황은 따로 내릴 수 있습니다.).
--   10. 우측상단은 Osu!주크박스로, 무작위한 순서로 노래가 재생됩니다.
--   11. 상단중앙에는 비트맵의 수, 오스!의 실행시간과 현재 컴퓨터의 시간을 볼 수 있습니다.
--   12. 좌측상단에는 사용자패널이 있습니다. 누르면 사용자설정이 나옵니다.
-
-## **사용자설정**
-
-<img src="useroptions1.png" title="유저 옵션" alt="유저 옵션" width="280" />
+<!--Logo definitions-->
+[Osu!]: /wiki/shared/mode/osu.png
+[Taiko]: /wiki/shared/mode/taiko.png
+[Ctb]: /wiki/shared/mode/catch.png
+[Mania]: /wiki/shared/mode/mania.png
+[Heart]: /wiki/shared/icon/heart.gif
+[Fire]: /wiki/shared/icon/flame.gif
+
+# 인터페이스
+
+![인터페이스 스크린](img/Intro_static.jpg)
+
+## 메인 메뉴
+
+![메인 메뉴](img/Introscreen.jpg)
+
+- \[1\] [오스!](/wiki/Glossary/#cookie?locale=ko) 로고는 재생되고 있는 노래의 설정된 [BPM](/wiki/BPM?locale=ko)에 맞춰 뜁니다(둠칫둠칫!).
```

Comment:
```suggestion
- \[1\] [오스!](/wiki/Glossary/#cookie) 로고는 재생되고 있는 노래의 설정된 [BPM](/wiki/BPM)에 맞춰 뜁니다(둠칫둠칫!).
```
The `?locale=ko` part in the link is not needed, the website will handle this. Please remove it from all of the other wiki links.

Reply:
ok, i've done this